src/main/kotlin/com/hibob/academy/employeeFeedback/service/AuthenticationService.kt

```diff
@@ -0,0 +1,46 @@
+package com.hibob.academy.employeeFeedback.service
+
+import com.hibob.academy.employeeFeedback.dao.EmployeeDao
+import com.hibob.academy.employeeFeedback.dao.LoggedInEmployeeInfo
+import com.hibob.academy.employeeFeedback.dao.LoginParams
+import io.jsonwebtoken.Jwts
+import io.jsonwebtoken.SignatureAlgorithm
+import org.springframework.stereotype.Component
+import java.time.LocalDateTime
+import java.time.ZoneOffset
+import java.util.*
+
+@Component
+class AuthenticationService(private val employeeDao: EmployeeDao) {
+
+    companion object{
+        const val SECRET_KEY = "kgjhdkjgbnsdjgbejr43775384957934579fhdjbdjsnks"
+    }
+
+    fun authenticate(id: Int, firstName: String, lastName: String): LoggedInEmployeeInfo {
+        val loginParams = LoginParams(id, firstName, lastName)
+        val loggedInEmployeeInfo = employeeDao.findEmployeeByLoginParams(loginParams)
+            ?: throw Exception("Authentication failed")
```

Comment:
Make sure that the user gets a "Not authorized" / "No access" exception, not an internal server error.
It's more user friendly